src/styles/grid.st.css

```diff
@@ -0,0 +1,56 @@
+/**
+ * = GRID - grid.st.css 
+ *
+ * Our main grid styles live here, you maybe tempted to make add a load of
+ * utilities that configure your grid in a load of ways... Be warned that having
+ * too much layout specified with the markup via props/classnames is a real pain 
+ * when you are trying to make changes to a page full of grids. Opinion.
+ * 
+ * ========================================================================= */
+
+
+/* = Default project vars. */
+:import {
+  -st-from: "@actionishope/shelley/styles/default/project.st.css";
+  -st-named:
+    --spacing-unit,
+    breakpoint-sm;
+}
+
+/* = Shelley project scope and vars. */
+:import {
+  -st-from: "./project.st.css";
+  -st-default: Shelley;
+}
+
+/* = Root Component */
+:import {
+  -st-from: "@actionishope/shelley/components/Grid/grid.st.css";
+  -st-default: Grid;
+}
+
+@st-scope Shelley {
+
+  /* = Your grid styles here... */
+
+  /**
+   * Grid:variant(1 - 4) have been assigned as part of our default.
+   * You can overide those in your project file. Just import them 
+   * from our default and override.
+   */
+
+  /* = Variant 5: ratio 3 : 7 */
+
+  Grid:variant(5) {
+    display: grid;
+    grid-column: goal;
+  }
+}
+
+@media value(breakpoint-sm) {
+  @st-scope Shelley {
+    Grid:variant(5) {
```

Comment:
A quick grid example of how you can extend/override Shelley grid styles for commonly used grid patterns throughout the site.

Reply:
CSS grid is the most amazing leap forward and it's easy to use. The grid systems that come with some frameworks just bends my mind and you end up with some much code you never use. CSS literally has a grid system built in and Shelley really promotes that.